sdk/core/Azure.Core/src/Serialization/ModelSerializerOptions.cs

```diff
@@ -12,32 +12,28 @@ namespace Azure.Core.Serialization
     public class ModelSerializerOptions
     {
         /// <summary>
-        /// Initializes a new instance of the <see cref="ModelSerializerOptions" /> class. Takes in a string that determines Format of serialized model. "D" = data format which means IgnoreReadOnly and IgnoreAdditionalProperties are false, "W" = wire format which means both properties are true. Default is "D".
+        /// Initializes a new instance of the <see cref="ModelSerializerOptions" /> class using a default for of <see cref="ModelSerializerFormat.Json"/>.
         /// </summary>
-        /// <param name="format"></param>
-        public ModelSerializerOptions(string format = "D")
+        public ModelSerializerOptions()
+            : this(ModelSerializerFormat.JsonValue) { }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ModelSerializerOptions" /> class.
+        /// </summary>
+        /// <param name="format"> The format to serialize to and deserialize from. </param>
+        public ModelSerializerOptions(ModelSerializerFormat format)
         {
-            //throw ArgumentException if not "D" or "W"
-            Format = ValidateFormat(format);
+            Format = format;
```

Comment:
It still would be possible to pass a random value for format here - should we leave the validation in place?

Reply:
random value is by design so people who implement our interface can come up with their own formats.  Each model will throw if it receives a format it doesn't understand.